# rolldown-resolve-issue-repo1

This repository reproduces the problem of the path of the id returned by `PluginContext.resolve` is incorrect.

